packages/core/src/Stores/tradinghub-store.js

```diff
@@ -14,4 +15,10 @@ export default class TradingHubStore extends BaseStore {
     toggleIsTourOpen(is_tour_open) {
         this.is_tour_open = is_tour_open;
     }
+    setIsOnboardingVisited() {
```

Comment:
we can merge `setIsOnboardingVisited` and `unsetIsOnboardingVisited` together
```suggestion
    setIsOnboardingVisited(is_visited) {
    this.is_onboarding_visited = is_visited
    }
```

Reply:
We mostly use this style of code for toggling, I think having separated function for making the value true or false is the better approach since we already have this structure in our code base.

Reply:
can you please share the approach you've in the codebase?
I found this approach 
![image](https://user-images.githubusercontent.com/82078941/206175742-7a281f5c-1740-4dc3-90cf-8136f6f8d6b0.png)